api/package.json

```
@@ -30,6 +30,7 @@
     "@aws-sdk/client-dynamodb": "3.896.0",
     "@aws-sdk/client-lambda": "3.896.0",
     "@aws-sdk/client-s3": "3.896.0",
+    "@aws-sdk/client-ses": "^3.922.0",
```

we want to remove the carrot here `^` to match with the 
```suggestion
    "@aws-sdk/client-ses": "3.922.0",
```